packages/test-utils/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@celo/test-utils",
+  "version": "0.0.1",
+  "description": "Celo common test and other dev utilities",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "homepage": "https://github.com/celo-org/celo-monorepo/tree/master/packages/test-utils",
+  "repository": "https://github.com/celo-org/celo-monorepo/tree/master/packages/test-utils",
+  "scripts": {
+    "build": "tsc -b .",
+    "clean": "tsc -b . --clean",
+    "lint": "tslint -c tslint.json --project ."
+  },
+  "dependencies": {},
```

Comment:
we should add `jest` and any other library we use here as a dependency (not dev) since it's a dev package

Reply:
Do we still need it here if it's in the top level `package.json`?

Reply:
yeah, good question.
I would say "yes"... and maybe in the future favor other packages depend on dev-utils and thanks to that obtain jest as a dependency.